dataverk_airflow/kubernetes_operator.py

```diff
@@ -33,29 +34,31 @@ class MissingValueException(Exception):
 def kubernetes_operator(
         dag: DAG,
         name: str,
-        repo: str,
         image: str,
-        cmds: list | None = None,
+        repo: str = None,
+        is_composer: bool = False,
```

Comment:
Hvorfor må brukere forholde seg til dette, kan vi ikke sjekke dette internt mot bucket-miljøvariabelen, eller noe lignende? Gjelder for alle de andre operatorene også.